src/main/java/com/bench/project/controller/DefaultController.java

```diff
@@ -1,40 +1,35 @@
 package com.bench.project.controller;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
-import org.springframework.amqp.core.AmqpTemplate;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.amqp.core.TopicExchange;
 import org.springframework.amqp.rabbit.annotation.EnableRabbit;
 import org.springframework.amqp.rabbit.annotation.RabbitListener;
+import org.springframework.amqp.rabbit.core.RabbitTemplate;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.GetMapping;
 import org.springframework.web.bind.annotation.PathVariable;
 import org.springframework.web.bind.annotation.RestController;
 
-import static com.bench.project.config.RabbitConfiguration.queueName;
+import static com.bench.project.config.RabbitConfiguration.DEFAULT_ROUTING_KEY;
 
 @RestController
 @EnableRabbit
+@Slf4j
+@RequiredArgsConstructor
 public class DefaultController {
 
-    @RabbitListener(queues = queueName)
-    public void processQueue(String message) {
-        logger.info("New message " + message);
-    }
-
-    private final Logger logger = LoggerFactory.getLogger(DefaultController.class);
-
-    private final AmqpTemplate template;
+    @Autowired
+    private final RabbitTemplate template;
 
     @Autowired
-    public DefaultController(AmqpTemplate template) {
-        this.template = template;
-    }
+    private final TopicExchange exchange;
 
     @GetMapping("/hello/{message}")
     public ResponseEntity<String> hello(@PathVariable String message) {
-        logger.info("Receive message: " + message);
-        template.convertAndSend(queueName, message);
+        log.info("Receive message to default queue: " + message);
+        template.convertAndSend(exchange.getName(), DEFAULT_ROUTING_KEY, message);
```

Comment:
i'm not sure but DEFAULT_ROUTING_KEY in this construction will not work correctly